spark/dl/src/main/scala/com/intel/analytics/bigdl/models/inception/Options.scala

```diff
@@ -39,7 +39,8 @@ object Options {
     warmupEpoch: Option[Int] = None,
     gradientL2NormThreshold: Option[Double] = None,
     gradientMin: Option[Double] = None,
-    gradientMax: Option[Double] = None
+    gradientMax: Option[Double] = None,
+    visualization: Boolean = true
```

Comment:
Can we change the ```visualization:Boolean``` to something like ```summary:Option[String]``` and let user input the visualization summary location? So if user inputs such value, we can save the graph visualization file in that location instead of hard code it in our code; if user doesn't input, it means he/she doesn't want to enable the visualization